TransferMethod/Sources/ListTransferMethodController.swift

```diff
@@ -32,8 +32,6 @@ final class ListTransferMethodController: UITableViewController {
 
     private lazy var emptyListLabel: UILabel = view.setUpEmptyListLabel(text: "emptyStateAddTransferMethod"
         .localized())
-    private lazy var addAccountButton: UIButton =
-        view.setUpEmptyListButton(text: "mobileAddTransferMethodHeader".localized(), firstItem: emptyListLabel)
```

Comment:
Can you check if func setUpEmptyListButton is used anywhere else. If not then you can cleanup the code by deleting this function as well

Reply:
Checked, it is not used anywhere else. So removed it.